src/solvers/euler.pyx

```diff
@@ -103,6 +103,7 @@ cdef class ImplicitEuler(Explicit_ODE):
             self.event_func = event_func
             self._event_info = N.array([0] * self.problem_info["dimRoot"]) 
             self.g_old = self.event_func(self.t, self.y)
+            self.statistics["nstatefcns"] += 1
```

Comment:
Should it really by nstatefcns?

Reply:
In ode.pyx it is defined by

self.statistics.add_key("nstatefcns", "**Number of state function evaluations**")

So, yes.